slsim/Util/param_util.py

```diff
@@ -448,3 +448,16 @@ def vel_disp_from_m_star(m_star):
     """
     v_disp = np.power(10, 2.32) * np.power(m_star / 1e11, 0.24)
     return v_disp
+
+
+def galaxy_size_redshift_evolution(z):
+    """This function provides a galaxy size which evolves with the redshift. This
+     relation is taken from https://iopscience.iop.org/article/10.1088/0067-0049/219/2/15/pdf
```

Comment:
I would put the Author et al in the description as well and link to the arXiv landing page, not the pdf.
Also, the description of the routine is not accurate. This is specifically a correction factor on the size relative to z=0